IdentityCore/src/broker_operation/response/MSIDBrokerOperationAccountResponse.m

```diff
@@ -0,0 +1,65 @@
+//
+//  MSIDBrokerOperationAccountResponse.m
+//  IdentityCore iOS
+//
+//  Created by JZ on 10/1/19.
+//  Copyright © 2019 Microsoft. All rights reserved.
+//
+
+#import "MSIDBrokerOperationAccountResponse.h"
+#import "MSIDAccount+MSIDJsonSerializable.h"
+
+@implementation MSIDBrokerOperationAccountResponse
+
+#pragma mark - MSIDJsonSerializable
+
+- (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)error
+{
+    self = [super initWithJSONDictionary:json error:error];
+    
+    if (self)
+    {
+        if (![json msidAssertType:NSDictionary.class
```

Comment:
There is a shorter version:

if (![json msidAssertType:NSDictionary.class ofKey:@"response_data" required:YES error:error]) return nil;

Reply:
:+1: addressed :)